### DISCUSSION TITLE
Problem with installation Wizard: can't connect to https://api.ravendb.net

### DISCUSSION THREAD

**Discussion by damienhocking** [ravendb/ravendb#15201] [General]
Hi, we use Raven in production and we love it. I'm doing a small experimental project on an Ubuntu 2022 VM (it has its own static IP), and for the life of me I can't get the wizard to get past the license key step. I've removed and reinstalled .NET 6, I cleaned and re-trusted the dotnet dev-certs and I can't get this to work. I've disabled the firewall. This is what I get:

![image](https://user-images.githubusercontent.com/1317330/199366170-bb787687-d5a7-457a-9571-1bf996f28d65.png)

Can anyone help? This is going to be something embarrassingly simple I'm sure.

**Comment by gregolsky** [comment]
Can you try checking if your time and date is set correctly on that machine?
And if your `ca-certificates` package is up to date?

**Reply by ayende** [reply]
Also, you are very likely behind some sort of a proxy / firewall that intercepts such calls.
Can you check what happens if you use `curl https://api.ravendb.net ` ?

**Reply by damienhocking** [reply]
I tried the Windows version on a VM running on the same VM server and it immediately works. The certs are updated on the Ubuntu VM, and I did a curl -v https://api.ravendb.net and the connection and certificates are good. I'm thinking this must be something specific to the .NET 6 runtime on the Ubuntu VM. Is there a way to force .NET to trust your server certificate?

**Reply by ayende** [reply]
The `ca-certificates` is the typical tool that you use for that.

**Reply by damienhocking** [reply]
Something's broken on that particular VM. I just tried a new installation of 22.04 LTS into a VM, installed dotnet6 and RavenDB and it immediately connects and activates. We can lift and shift the workloads from the old VM easily to the new one. Thanks for the help.

**Comment by mposch** [comment]
Hello Community!

I have a similar issue. I have an installation that is only able to connect to the internat via a proxy. When i export the proxy´s hostname and port to the http_proxy variable, i am able to connect via curl:
````
 curl -v https://api.ravendb.net
....
> CONNECT api.ravendb.net:443 HTTP/1.1
> Host: api.ravendb.net:443
....
> Proxy-Connection: Keep-Alive
>
< HTTP/1.1 200 Connection established
```` 
How can i tell the ravendb to use this proxy as well ? Is there a config setting for a proxy?

Thanks,

Matthias

**Reply by ayende** [reply]
You should be able to define `http_proxy` for ravendb as well
